nixos/modules/virtualisation/oci-containers.nix

```diff
@@ -221,6 +221,13 @@ let
           example = "hello-world";
         };
 
+        preRunExtraOptions = mkOption {
+          type = with types; listOf str;
+          default = [];
+          description = "Extra options for {command}`${defaultBackend}` that go before the `run` argument.";
+          example = ["--runtime runsc"];
```

Comment:
Will this work as-is after being passed through `escapeShellArg`?  It looks like the backend will get `--runtime\ runsc` as a single argument, and I suspect the example would need to be either `["--runtime=runsc"]` or `["--runtime" "runsc"]`.